### ISSUE TITLE
AttributeError: 'ViewAnalyzer' object has no attribute 'project'

### ISSUE THREAD

**Issue by TheBellman** [servian/bigquery-view-analyzer#25]
Executing against our views, fails out quickly (project name and view name suppressed below)

```
% bqva tree --status --no-key --view "PROJECT:analytics_staging.TABLE"

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/bin/bqva", line 8, in <module>
    sys.exit(main())
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/bqva/cli.py", line 90, in tree
    formatted_tree = utils.format_tree(view, show_status=status, show_key=key)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/bqva/utils.py", line 33, in format_tree
    output.append(pre + format_node(node, show_status=show_status))
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/bqva/utils.py", line 38, in format_node
    project = color(Fore.CYAN, node.project)
AttributeError: 'ViewAnalyzer' object has no attribute 'project'
```

**Comment by Marty08** [comment]
Getting the same message too.

**Comment by christippett** [comment]
Sorry folks, seems I broke things with my last update. I've just released `21.9.0` that should hopefully resolve this.

**Comment by Marty08** [comment]
updated to version 21.9.0 and can confirm no issues
